docs/Account/signers/passport.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_label: "Passport"
+sidebar_position: 9
+---
+
+# Passport
+
+[Passport](https://docs.0xpass.io) is an MPC-based programmable, distributed, and non-custodial key management system, that allows users to generate wallets, scoped to their application, either via user Passkeys or any developer defined authentication method. Our signer allows you to sign messages and transactions with a Passport Network account.
+
+## Dependencies
+
+You will need the following dependencies to create a Smart Account this way:
+
+```bash
+yarn add @biconomy/account viem@2.9.4 @0xpass/passport @0xpass/webauthn-signer @0xpass/passport-viem @0xpass/key-signer
+```
+
+## Imports
+
+```typescript
+import { Passport, TESTNET_RSA_PUBLIC_KEY } from "@0xpass/passport";
+import { WebauthnSigner } from "@0xpass/webauthn-signer";
+import { KeySigner } from "@0xpass/key-signer";
+import { createPassportClient } from "@0xpass/passport-viem";
+import { http } from "viem";
+import { sepolia } from "viem/chains";
+import { createSmartAccountClient } from "@biconomy/account";
+```
+
+## Passport Configuration
+
+To use Passport, you'll first need to make sure you have configured a scope for your application. For this you can follow the guides below:
+
+- Refer to the [Passport documentation](https://docs.0xpass.io/) for instructions on setting up your application with Passport.
+- For a primer on setting up your scope you can check [here](https://docs.0xpass.io/authentication/configuring-your-scope).
+
+## Passkey Signer
+
+Passport offers a Passkey signer, which follows the webauthn standard, which is a user owned authentication method that allows users to sign messages and transactions with their Passkey.
```

Comment:
can we add reference to https://docs.0xpass.io/authentication/passkeys/registration/register-using-sdk here, given details like rpId, rpName are not self explanatory.

Reply:
Done!